### DISCUSSION TITLE
【Fabric】 2-2.9.80+ex119-1.19.2-fabric	启动崩溃

### DISCUSSION THREAD

**Discussion by PiPiTcc** [IndustrialCraftMC/IndustrialCraft2#15] [【漏洞反馈】Bugs Report]
[当前fabric版最新工业2CI](http://jenkins.ic2.player.to/job/IC2/job/1.19/lastSuccessfulBuild/artifact/tmp/out/industrialcraft-2-2.9.80+ex119-1.19.2-fabric.jar)

Minecraft Version: Java 1.19.2
Fabric Version: 0.14.19
Fabric API Version: 0.76.0+1.19.2

![_4$}37E3H_2LSG%)CB92F%T](https://user-images.githubusercontent.com/47528293/236349303-6a05ee3d-2a1f-4553-ae5d-463ee927788d.png)
[minecraft-exported-logs-2023-05-05T07-04-57.log](https://github.com/AbbottMc/IndustrialCraft2/files/11402264/minecraft-exported-logs-2023-05-05T07-04-57.log)

**Comment by AbbottMc** [comment]
这个要等Player修构建站的脚本

**Comment by AbbottMc** [comment]
已修复
